run.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
-if [ "$1" == "index" ]
```

Comment:
This script shouldn't have changed right? Just the version in `./.docker`?

Reply:
Yeah - will revert this change.

Although I think @ltalirz simply wanted to remove this way of choosing to run the index meta-database, and instead rely on the environment variable `MAIN` to be set - but I see no reason why we can't have both, having the passing of `index` to `run.sh` override the `MAIN` environment variable. Right?

Reply:
I'd be fine with that.